ResidentsSocialCarePlatformApi.Tests/V1/Factories/ResponseFactoryTests.cs

```diff
@@ -142,18 +142,18 @@ public void CanMapSummarisedCaseNoteInformationFromDomainToResponse()
                     MosaicId = "12345",
                     CaseNoteId = 67890,
                     CaseNoteTitle = "I AM A CASE NOTE",
-                    EffectiveDate = recordOneTime,
-                    CreatedOn = recordOneTime,
-                    LastUpdatedOn = recordOneTime
+                    EffectiveDate = recordOneTime.ToString("s"),
+                    CreatedOn = recordOneTime.ToString("s"),
+                    LastUpdatedOn = recordOneTime.ToString("s")
```

Comment:
For clarity, I think our test should explicitly show what the formatted date looks like e.g. `2021-03-01T15:00` instead of calling `.ToString("s")` in our test which is also used in the production code. What do you think?

Reply:
Btw, don't worry about this now! Feel free to merge this in and refactor in a follow-up PR as deploying to production is our priority.